helper/date_format.go

```diff
@@ -22,6 +22,20 @@ func DateFormat(s string) string {
 	return template.HTMLEscapeString(t.Format("02 January 2006"))
 }
 
+// TimeFormat extracts the BST time value for 12hr clock from ISO8601 formatted timestamps
+func TimeFormat(s string, is24HRs bool) string {
+	t, err := time.Parse(time.RFC3339, s)
+	if err != nil {
+		log.Error(context.Background(), "failed to parse time", err)
+		return template.HTMLEscapeString(s)
+	}
+	t = localiseTime(&t)
+	if !is24HRs {
+		return template.HTMLEscapeString(t.Format("03:04pm"))
+	}
+	return template.HTMLEscapeString(t.Format("15:04"))
```

Comment:
It seems a bit counterintuitive if we negate the condition in the if. Maybe change it to not be negative for easier reading?

```suggestion
	if is24HRs {
		return template.HTMLEscapeString(t.Format("15:04"))
	}
	return template.HTMLEscapeString(t.Format("03:04pm"))
```

Reply:
because we've split the function now. we don't need to worry about this comment. However i agree with the suggestion.